### ISSUE TITLE
Bug: Vue or React routers are broken on direct entry pathing 

### ISSUE THREAD

**Issue by cghiurea** [devforth/spa-to-http#15]
Version: 1.0.7

Create a react or vue spa with a router.  If you navigate to the router it will return a 404 with your new change of "prevent path traversal outside content directory"

Context our setup on our dev server is Apache reverse proxy to Spa-to-http to React Spa.
The second setup is Traefik to Spa-to-HTTP to Vue Spa.

It returns a 404 .

![image](https://github.com/devforth/spa-to-http/assets/148401030/24dd4b95-07c4-4269-8511-118d36e8e3b8)

**Comment by Satont** [comment]
Yep, on 1.0.6 version it is working, just tested.

**Comment by cghiurea** [comment]
@Satont yes, I forgot to mention version 1.0.6 it works. I had to revert to that version on our production servers.

**Comment by sunaley** [comment]
> @Satont yes, I forgot to mention version 1.0.6 it works. I had to revert to that version on our production servers.

I think this might be due to fixing this issue #14 .

**Comment by cghiurea** [comment]
It is 100% because it tries to reroute and does not work.

**Comment by ivictbor** [comment]
This was fixed in `1.0.8` 8 month ago. 

Just retested:

```
npm create vue@latest
// set yes in: ✔ Add Vue Router for Single Page Application development? … No / Yes

cd vue-project
npm install

cat <<EOF > Dockerfile
FROM node:20-alpine as builder
WORKDIR /code/
ADD package-lock.json .
ADD package.json .
RUN npm ci
ADD . .
RUN npm run build

FROM devforth/spa-to-http:latest
COPY --from=builder /code/dist/ .
EOF

docker build -q . | xargs docker run --rm -p 8080:8080
```

Got to http://localhost:8080/about and F5
